### PR TITLE
feat: Update browser api docs for npm

### PR DESCRIPTION
**Pull request by patrickhousley**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

This PR updates the callout at the top of the browser api docs pages to include a note about which agent feature is needed for the api to function when using the npm package.

https://issues.newrelic.com/browse/NR-141636

### CODE REVIEW

**Review by metal-messiah — APPROVED** [review]

**Comment by akristen** [comment]
Hi @metal-messiah -- want to confirm something with our docs Browser liaison, then get back to these edits. I'm mostly curious if there's a better place to give this tip + some formatting/styling questions (since this edit touches 22 docs, want to make sure it's all good so we don't have to go back and edit all at once again)

File: src/content/docs/browser/new-relic-browser/browser-apis/actiontext.mdx [hunk 1]
```diff
@@ -13,8 +13,8 @@ redirects:
 ---
 
 <Callout variant="important">
-To use this API, you need the Browser Pro+SPA edition of the browser agent.
-  </Callout>
+To use this API, you need the Browser Pro+SPA edition of the browser agent. If you are using the NPM package, you will need to include the `spa` feature.
```
Comment (ally-sassman) [inline src/content/docs/browser/new-relic-browser/browser-apis/actiontext.mdx:16]:
```suggestion
To use this API, you need the Browser Pro+SPA edition of the browser agent. If you're using the [NPM package to install the browser agent](/docs/apis/nerdgraph/examples/combining-npm-nerdgraph/), you must include the `spa` feature.
```
@patrickhousley How do they "include the `spa` feature"? Let's add a bit more detail or linked instructions on how to do this.

**Comment by patrickhousley** [comment]
Its documented in the npm package itself. Sorry, out sick today but you can
link the the readme in the browser agent repo or the npmjs.org page, just
search for newrelic browser agent.

File: src/content/docs/browser/new-relic-browser/browser-apis/setcustomattribute.mdx [hunk 2]
```diff
@@ -17,8 +17,8 @@ redirects:
 ---
 
 <Callout variant="important">
-This API will work for any browser edition (Browser Lite, Pro, or Pro+SPA).
-</Callout>  
+This API will work for any browser edition (Browser Lite, Pro, or Pro+SPA). If you are using the NPM package, you will need to include at least one feature.
```
Comment (ally-sassman) [inline src/content/docs/browser/new-relic-browser/browser-apis/setcustomattribute.mdx:20]: @patrickhousley  What does "include at least one feature" mean?

Comment (patrickhousley) [reply src/content/docs/browser/new-relic-browser/browser-apis/setcustomattribute.mdx:20]: When creating an instance of the agent from the NPM package, at least one feature must be included for this API to function.

Comment (ally-sassman) [reply src/content/docs/browser/new-relic-browser/browser-apis/setcustomattribute.mdx:20]: @patrickhousley Like do you mean any monitoring feature? What's an example of this?

Comment (patrickhousley) [reply src/content/docs/browser/new-relic-browser/browser-apis/setcustomattribute.mdx:20]:
```
import { Agent } from '@newrelic/browser-agent/loaders/agent'
import { Metrics } from '@newrelic/browser-agent/features/metrics'
import { PageViewEvent } from '@newrelic/browser-agent/features/page_view_event'
import { PageViewTiming } from '@newrelic/browser-agent/features/page_view_timing'

const options = {
  info: { ... },
  loader_config: { ... },
  init: { ... }
}

new Agent({
  ...options,
  features: [
    // These are the features, at least one feature needs to be in this list, doesn't matter which one.
    Metrics,
    PageViewEvent,
    PageViewTiming
  ]
})
```

**Pull request merged**